chalice/app.py

```diff
@@ -389,7 +394,12 @@ def raw_body(self):
 
     @property
     def json_body(self):
-        if self.headers.get('content-type', '').startswith('application/json'):
+        content_type = self.headers.get('content-type', '')
+        acceptable = any([
+            True for acceptable_type
+            in self.ACCEPTABLE_JSON_CONTENT_TYPES
+            if content_type.startswith(acceptable_type)])
+        if acceptable:
```

Comment:
Could probably go with `content_type in self.ACCEPTABLE_JSON_CONTENT_TYPES` and make the `ACCEPTABLE_JSON_CONTENT_TYPES` a frozenset.

Reply:
That would be cleaner but wouldn't work if the `Content-Type` was `application/json; charset=utf-8` :-(